common/build.rs

```diff
@@ -1,8 +1,8 @@
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    println!("cargo:rerun-if-changed=proto");
+    println!("cargo:rerun-if-changed=../proto");
 
     tonic_build::configure()
         .out_dir("codegen/grpc")
-        .compile(&["proto/helloworld.proto"], &["proto"])?;
+        .compile(&["../proto/helloworld.proto"], &["../proto"])?;
```

Comment:
```suggestion
        .build_client(false)
        .compile(&["../proto/helloworld.proto"], &["../proto"])?;
```
I suppose now it's safe to say we're don't generate`tonic` clients